product_docs/docs/efm/4/05_using_efm.mdx

```diff
@@ -39,6 +39,8 @@ To start the Failover Manager cluster on RHEL/CentOS 7.x or RHEL/Rocky Linux/Alm
 
 `systemctl start edb-efm-4.<x>`
 
+!!! Note
+    If the agent fails to start, see the startup log `/var/log/efm-4.<x>/startup-efm.log` for more information.
```

Comment:
I used the 4.<x> format here to be consistent with the page, but a more general efm-<version> format on the troubleshooting page.